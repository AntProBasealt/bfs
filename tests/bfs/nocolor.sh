bfs_diff rainbow -nocolor
