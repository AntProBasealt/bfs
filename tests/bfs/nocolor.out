rainbow
rainbow/broken
rainbow/chardev_link
rainbow/exec.sh
rainbow/file.dat
rainbow/file.txt
rainbow/link.txt
rainbow/mh1
rainbow/mh2
rainbow/ow
rainbow/pipe
rainbow/sgid
rainbow/socket
rainbow/star.gz
rainbow/star.tar
rainbow/star.tar.gz
rainbow/sticky
rainbow/sticky_ow
rainbow/sugid
rainbow/suid
